OptimizelySDK/Odp/OdpConfig.cs

```diff
@@ -0,0 +1,75 @@
+﻿/* 
+ * Copyright 2022, Optimizely
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System.Collections.Generic;
+
+namespace OptimizelySDK.Odp
+{
+    public class OdpConfig
+    {
+        /// <summary>
+        /// Public API key for the ODP account from which the audience segments will be fetched (optional).
+        /// </summary>
+        public string ApiKey { get; private set; }
```

Comment:
Don't we need to lock-protect all these "ApiKey", "ApiHost" and "SegmentsToCheck" get/set for thread-safety?